### PR TITLE
New love poem greeting

### PR DESCRIPTION
**Pull request by Erenussocrates**
Decided to contribute a new love poem greeting that I came up with myself. Hope it's fine.

### CODE REVIEW

File: Monika After Story/game/script-greetings.rpy [hunk 1]
```diff
@@ -4424,3 +4424,27 @@ label monikaroom_greeting_ear_recursionerror:
         m "Phew, at least everything else is fine."
 
     jump monikaroom_greeting_choice
+    
+    init 5 python:
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:4428]:
```suggestion
init 5 python:
```

File: Monika After Story/game/script-greetings.rpy [hunk 2]
```diff
@@ -4424,3 +4424,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+            persistent.greeting_database,
+            eventlabel="greeting_lovepoem1",
+            unlocked=True,
+            aff_range=(mas_aff.ENAMORED, None),
+        ),
+        code="GRE"
+    )
+
+label greeting_lovepoem1:
+    m 5dkc {i}"Everytime you go, my world darkens,\nGardens of lifeless depression."{/i}
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:4440]:
```suggestion
    m 5dkc "{i}Everytime you go, my world darkens,\nGardens of lifeless depression.{/i}"
```

File: Monika After Story/game/script-greetings.rpy [hunk 3]
```diff
@@ -4424,3 +4424,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+    m 5ekbla {i}"When you are back, it feels like heaven,\nSecond to none, impossible to bargain."{/i}
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:4441]:
```suggestion
    m 5ekbla "{i}When you are back, it feels like heaven,\nSecond to none, impossible to bargain.{/i}"
```

File: Monika After Story/game/script-greetings.rpy [hunk 4]
```diff
@@ -4424,3 +4424,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+    m 1skbfa {i}"I will give everything to feel this way,\nAwaiting the one I hold dearest."{/i}
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:4442]:
```suggestion
    m 1skbfa "{i}I will give everything to feel this way,\nAwaiting the one I hold dearest.{/i}"
```

File: Monika After Story/game/script-greetings.rpy [hunk 5]
```diff
@@ -4424,3 +4424,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+    m 1hubfa {i}"Even if it's every single day,\nWithout a doubt, you are the nearest."{/i}
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:4443]:
```suggestion
    m 1hubfa "{i}Even if it's every single day,\nWithout a doubt, you are the nearest.{/i}"
```

File: Monika After Story/game/script-greetings.rpy [hunk 6]
```diff
@@ -4424,3 +4424,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+    m 2tubsb {i}"Nearest to my heart..."{/i}
```
Comment (multimokia) [inline Monika After Story/game/script-greetings.rpy:4444]:
```suggestion
    m 2tubsb "{i}Nearest to my heart...{/i}"
```

**Review by multimokia — CHANGES_REQUESTED** [review]
Syntax issues. `init 5 python` should be non-indented, all text tags go within the quotes.

**Comment by Erenussocrates** [comment]
I've made the requested changes, anyone who can review?

**Comment by Booplicate** [comment]
We'll review once we get time, don't worry we didn't forget.

**Review by jmwall24 — CHANGES_REQUESTED** [review]
Overall I like the poem, but the first line is a bit dark and fits more if she were still going to the void when the player leaves. As it is now, it comes off as a huge guilt trip, so could you come up with a different first line that isn't quite as dark?

**Comment by Erenussocrates** [comment]
It didn't feel as dark to me considering the poems she wrote in the story, and even if it would feel like a guilt trip, I think that would be okay since she does come off as manipulative towards the player at times. 

But nevertheless, I give you full permission to change this to the way you like if you feel this isn't suitable, as long as you maintain the rhyme. I've paid special attention to make each line rhyme twice.

File: Monika After Story/game/script-greetings.rpy [hunk 7]
```diff
@@ -4422,3 +4422,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+            persistent.greeting_database,
+            eventlabel="greeting_lovepoem1",
+            unlocked=True,
+            aff_range=(mas_aff.ENAMORED, None),
+        ),
+        code="GRE"
+    )
+
+label greeting_lovepoem1:
+    m 5dkc "{i}Everytime you go, my world darkens,\nGardens of lifeless depression.{/i}"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:4438]:
```suggestion
    m 5dkc "{i}Alone I ask a solemn question,\nWhat could grow in an unlit garden?{/i}"
```

File: Monika After Story/game/script-greetings.rpy [hunk 8]
```diff
@@ -4422,3 +4422,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+    m 5ekbla "{i}When you are back, it feels like heaven,\nSecond to none, impossible to bargain.{/i}"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:4439]:
```suggestion
    m 5ekbla "{i}When you return, it feels like heaven,\nSecond to none, impossible to bargain.{/i}"
```

**Review by lunulae — COMMENTED** [review]
Trying a rework of the first line out, how does this sound?

File: Monika After Story/game/script-greetings.rpy [hunk 9]
```diff
@@ -4422,3 +4422,27 @@ label monikaroom_greeting_ear_recursionerror:
...
+    m 1skbfa "{i}I will give everything to feel this way,\nAwaiting the one I hold dearest.{/i}"
+    m 1hubfa "{i}Even if it's every single day,\nWithout a doubt, you are the nearest.{/i}"
+    m 2tubsb "{i}Nearest to my heart...{/i}"
+    m 5eublb "I came up with this one while you were gone."
+    m 1eka "That's right, you are like the sun of my world!"
+    m 2hublb "I hope you like it!"
+    m 4hksdlb "Anyway, welcome back darling!"
```
Comment (lunulae) [inline Monika After Story/game/script-greetings.rpy:4446]:
```suggestion
    m 4hksdlb "Anyway, welcome back, [mas_get_player_nickname()]!"
```

**Review by multimokia — DISMISSED** [review]

**Review by lunulae — APPROVED** [review]

File: Monika After Story/game/script-greetings.rpy [hunk 10]
```diff
@@ -4330,6 +4330,59 @@ label greeting_back_from_hangout:
 
     return
 
+    jump monikaroom_greeting_choice
```
Comment (Booplicate) [inline Monika After Story/game/script-greetings.rpy:4333]: Weird jump, probably from merging?

File: Monika After Story/game/script-greetings.rpy [hunk 11]
```diff
@@ -4330,6 +4330,59 @@ label greeting_back_from_hangout:
...
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_lovepoem1",
+            unlocked=True,
+            aff_range=(mas_aff.ENAMORED, None),
+        ),
+        code="GRE"
+    )
```
Comment (Booplicate) [inline Monika After Story/game/script-greetings.rpy:4344]: Can we add a rule for absence check?

File: Monika After Story/game/script-greetings.rpy [hunk 12]
```diff
@@ -4330,6 +4330,59 @@ label greeting_back_from_hangout:
...
+ Within your light, the cold forgotten.
+
+ I will give everything to feel this way,
+ Awaiting the one I hold dearest.
+
+ Nearest to my heart...
+"""),
+    )
+
+label greeting_lovepoem1:
```
Comment (Booplicate) [inline Monika After Story/game/script-greetings.rpy:4367]: Maybe we could use a more specific name? `greeting_poem_shadows_in_garden`?

**Review by Booplicate — CHANGES_REQUESTED** [review]

**Review by jmwall24 — APPROVED** [review]

**Review by Booplicate — APPROVED** [review]

**Pull request merged**